nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/QueryDatabaseTable.java

```diff
@@ -184,23 +90,18 @@
             .addValidator(StandardValidators.NON_NEGATIVE_INTEGER_VALIDATOR)
             .build();
 
-
-    private final List<PropertyDescriptor> propDescriptors;
-
-    protected final Map<String, Integer> columnTypeMap = new HashMap<>();
-
     public QueryDatabaseTable() {
         final Set<Relationship> r = new HashSet<>();
         r.add(REL_SUCCESS);
         relationships = Collections.unmodifiableSet(r);
 
         final List<PropertyDescriptor> pds = new ArrayList<>();
         pds.add(DBCP_SERVICE);
+        pds.add(DB_TYPE);
         pds.add(TABLE_NAME);
         pds.add(COLUMN_NAMES);
         pds.add(MAX_VALUE_COLUMN_NAMES);
         pds.add(QUERY_TIMEOUT);
-        pds.add(SQL_PREPROCESS_STRATEGY);
```

Comment:
Why is the SQL_PREPROCESS_STRATEGY being removed from the QueryDatabaseTable processor?